pkg/api/const.go

```diff
@@ -319,12 +319,11 @@ const (
 	// AzureCniPluginVerLinux specifies version of Azure CNI plugin, which has been mirrored from
 	// https://github.com/Azure/azure-container-networking/releases/download/${AZURE_PLUGIN_VER}/azure-vnet-cni-linux-amd64-${AZURE_PLUGIN_VER}.tgz
 	// to https://kubernetesartifacts.azureedge.net/azure-cni
-	// AKS needs 1.0.33 for node local dns cache support
-	AzureCniPluginVerLinux = "v1.0.33"
+	AzureCniPluginVerLinux = "v1.1.2"
 	// AzureCniPluginVerWindows specifies version of Azure CNI plugin, which has been mirrored from
 	// https://github.com/Azure/azure-container-networking/releases/download/${AZURE_PLUGIN_VER}/azure-vnet-cni-windows-amd64-${AZURE_PLUGIN_VER}.zip
 	// to https://kubernetesartifacts.azureedge.net/azure-cni
-	AzureCniPluginVerWindows = "v1.0.33"
+	AzureCniPluginVerWindows = "v1.1.2"
 	// CNIPluginVer specifies the version of CNI implementation
 	// https://github.com/containernetworking/plugins
 	CNIPluginVer = "v0.7.6"
```

Comment:
do we want to update this?

Reply:
Think that's a different change we don't care about.
https://github.com/Azure/aks-engine/commit/1bbc94a1ccf7dde6236c303f6195fa3d7cfee3fe